### PR TITLE
added lambda layers resource

### PR DESCRIPTION
**Pull request by mrkenkeller**
This adds deleting lambda layers to aws nuke.

As per the contribution guidelines I used `ListLayersPages` and added a `properties` function.

### CODE REVIEW

File: resources/lambda-layers.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+
+	for _, layer := range layers {
+
+		var i int64 = int64(1) // is there a better way to do this
+		for i <= *layer.LatestMatchingVersion.Version {
```
Comment (svenwltr) [inline resources/lambda-layers.go:41]: Shouldn't that be a generic for loop? Something like:

```go
for i := 1; i < *layer.LatestMatchingVersion.Version; i++ {
```

File: resources/lambda-layers.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			resources = append(resources, &lambdaLayer{
```
Comment (svenwltr) [inline resources/lambda-layers.go:42]: I don't understand this. Does lambda always have *all* versions from 1 to `LatestMatchingVersion`?

File: resources/lambda-layers.go (hunk 1 above)
Comment (mrkenkeller) [reply resources/lambda-layers.go:41]: Sorry, old habits.  I'll fix and resubmit.

File: resources/lambda-layers.go (hunk 2 above)
Comment (mrkenkeller) [reply resources/lambda-layers.go:42]: It did not throw an error when it tried to delete a version that did not exist (I tested that), but on reflection the right way is to use the list provided by `listLayerVersions`.  I'll update and resubmit.

Comment (svenwltr) [reply resources/lambda-layers.go:42]: Thanks. It is not really about errors, but about performance for cases where there is a high version number.

**Review by svenwltr — APPROVED** [review]

**Pull request merged**